powa_collector/__init__.py

```diff
@@ -65,6 +68,7 @@ def __init__(self):
         signal.signal(signal.SIGTERM, self.sighandler)
 
     def connect(self, options):
+        """Connect to the repository"""
```

Comment:
We should mention here that this is the main connection to the repository, for all communication with powa-web or any user of the communication protocol, and that it's a persistent connection.